ztp/gitops-subscriptions/argocd/resource-hook-example/policygentemplates/common-ranGen.yaml

```diff
@@ -13,20 +13,19 @@ metadata:
 spec:
   bindingRules:
     common: "true"
+  complianceType: musthave
```

Comment:
I am thinking probably more clear if we add this under something like "policyDefaults"? so a default remediationType can be added here as well.
spec:
  policyDefaults:
    complianceType: musthave